tests/unit/util/CMakeLists.txt

```
@@ -3,5 +3,5 @@
 cmake_minimum_required(VERSION 3.20.0)
 
 project(util)
-set(SOURCES main.c maincxx.cxx ../../../lib/os/dec.c)
 find_package(Zephyr COMPONENTS unittest REQUIRED HINTS $ENV{ZEPHYR_BASE})
+target_sources(testbinary PRIVATE main.c maincxx.cxx ../../../lib/os/dec.c)
```

the rework in this PR allows you to use `ZEPHYR_BASE` instead of `../../../`

```suggestion
target_sources(testbinary PRIVATE main.c maincxx.cxx ${ZEPHYR_BASE}/lib/os/dec.c)
```

---

Done